readme.md

```diff
@@ -58,6 +58,19 @@ Minimum: `1`
 
 Concurrency limit.
 
+##### timeout
+
+Type: `number`
+
+Per-operation timeout in milliseconds. Operations fulfill once `timeout` elapses if they haven't already.
+
+##### throwOnTimeout
+
+Type: `boolean`<br>
+Default: `true`
+
+Whether or not a timeout is considered an exception.
+
```

Comment:
You also need to document the `.timeout` getter/setter.

Reply:
Ok should be good now, 
why are we duplicating the typings for documentation again though?